packages/apps/transfer/src/pages/faucet/existing/notification.tsx

```diff
@@ -0,0 +1,101 @@
+import {
```

Comment:
Would it make sense to put this in the Global components folder?

Reply:
Also discussed with @nil-amrutlal-dept , there are more parts to the pages which are reusable and could be abstracted. Would like to do this in a separate PR, also keeping in mind the new layout changes coming from @rezaerami